alerts/cluster-monitoring-operator/ThanosRuleQueueIsDroppingAlerts.md

```diff
@@ -0,0 +1,27 @@
+# ThanosRuleQueueIsDroppingAlerts
+
+## Meaning
+
+The [Thanos Ruler][1] is deployed when user workload monitoring is enabled. It
+allows alerting rules deployed as part of user workload monitoring to query both
+the the Prometheus instance responsible for cluster components as well the user
+workload Prometheus instance. This alert is triggered when the Thanos Ruler
+queue is found to be dropping alerting events.
+
+## Impact
+
+Alerts for user workloads may not be delivered.
+
+## Diagnosis
+
+Check the logs for the Thanos Rules pods:
```

Comment:
Are there any knowns we can specifically look for in those logs?

Reply:
Added some some sample output.